app/src/main/java/io/intrepid/skeleton/base/BaseFragment.java

```diff
@@ -0,0 +1,92 @@
+package io.intrepid.skeleton.base;
+
```

Comment:
This is something that clone projects can add if they see a need for it (or if too many projects do that, we can add it here).